examples/finance/bs_ivbp.ipynb

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
<p>You may want to switch to numpy rather than sympy to compute numerical values for <code>call_value_bs</code>   for example, sympy will be very slow when the problem gets bigger.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Done